recipe/meta.yaml

```
@@ -13,11 +15,10 @@ source:
   - path: setup.py
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
+  number: 2
   script_env:
     # Limit MAX_JOBS in order to prevent runners from crashing
-    - MAX_JOBS=4
+    - MAX_JOBS=20
     - TORCH_CUDA_ARCH_LIST=8.0;8.6;8.9;9.0+PTX
```

Reduce this to just `8.0+PTX` so builds are faster while we test things out.
```suggestion
    - TORCH_CUDA_ARCH_LIST=8.0+PTX
```

If you also uncomment the two lines at L27-28 (under the 'debugging skips below' comment), and re-render manually by running `conda smithy rerender -c auto` locally, that would help to conserve CI resources further until things are finalized.